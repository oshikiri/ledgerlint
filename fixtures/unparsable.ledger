; comment 1
; comment 2
XXX 2022/01/01 XXX
