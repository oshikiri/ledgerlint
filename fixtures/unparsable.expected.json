{"type":"diagnostic","source":"ledgerlint","file_path":"fixtures/unparsable.ledger","line_number":3,"range":{"start":{"line":2,"character":0},"end":{"line":2,"character":80}},"level":"WARN","severity":2,"message":"This line is neither comment nor header nor posting"}
